pyethapp/config.py

```diff
@@ -103,13 +115,14 @@ def _fix_accounts(path):
         write_config(config)
 
 
-def load_config(path=default_config_path):
+def load_config(path=default_config_path, fix_accounts=True):
```

Comment:
That is a good question. I asked yesterday on the channel but got no response. This flag is to call ```_fix_accounts()``` or not.

https://github.com/ethereum/pyethapp/blob/develop/pyethapp/config.py#L93

Checked what does `_fix_accounts` do and I can see that if this is present in ```config.yaml```:
```
eth:
  privkey_hex: 8087664c397d3b03153e3ebfb5dd9a21b6a2ce89a9fdee198dca73f19023df92
```

```config.yaml``` will be overwritten with:

```
accounts:
  privkeys_hex: [8087664c397d3b03153e3ebfb5dd9a21b6a2ce89a9fdee198dca73f19023df92]
```

Is it somehow related to that story? If not pls share what is a purpose of ```_fix_accounts```.

> note the `test_privkeys` quick option in hydra , which generates the alloc in genesis and creates unlocked keys for the accounts (both derived from the privkeys). note that the config must be in place when block 0 (genesis) is created and saved to disk. i.e. first invocation with the specified datadir. i guess this may  sound very confusing.